Source/WebCore/platform/BoxSides.h

```diff
@@ -274,7 +276,7 @@ constexpr BoxCorner mapCornerLogicalToPhysical(WritingMode writingMode, LogicalB
     return isLeft ? BoxCorner::BottomLeft : BoxCorner::BottomRight;
 }
 
-constexpr LogicalBoxCorner mapCornerPhysicalToLogical(WritingMode writingMode, BoxCorner boxCorner)
+constexpr LogicalBoxCorner mapCornerPhysicalToLogical(const WritingMode writingMode, const BoxCorner boxCorner)
```

Comment:
I don't really see the benefit here and not what we usually do in WebKit

Reply:
Should I revert these changes then? I put them in because it clarifies that these are only inputs and not outputs.

Reply:
Alan left it up to me, so I'm merging as-is.